docs/modules/ROOT/pages/vshn-managed/keycloak/customization.adoc

```diff
@@ -32,3 +32,133 @@ spec:
 <1> Container image url.
 <2> Secret name containing credentials for the registry.
 <3> Secret namespace
+
+
+== Configuration
+
+You can provide a custom configuration to keycloak using a `ConfigMap``.
+The content of the `ConfigMap` is a JSON file based on the keycloak export files.
+The configuration *must* be in the key `keycloak-config.json` inside the `ConfigMap`
+
+The configuration will be read during the startup of keycloak and loaded into the database.
+
+An extensive set of configuration examples can be found https://github.com/inventage/keycloak-config-cli/tree/main/src/test/resources/import-files[here^].
+
+The configuration also supports variable substition.
+
+Variables exposed by spring boot (through configtree or external configuration) can be accessed by `$(property.name)`
```

Comment:
```suggestion
Variables exposed by Spring Boot (through configtree or external configuration) can be accessed by `$(property.name)`
```

What is "configtree"? Are there any external links which describe these variables in more detail?

Reply:
This is a spring boot specific term: https://docs.spring.io/spring-boot/reference/features/external-config.html#features.external-config.files.configtree.